admin/controllers/availability.php

```diff
@@ -22,8 +22,22 @@ public static function announce()
     {
         if (userHasPermission('admin:shop:availability:manage')) {
 
+            //  Alerts
+            $alerts = array();
+            $ci     =& get_instance();
+
+            //  Get all notifications, show as an info count
+            $numAlertsAll = $ci->db->count_all_results(NAILS_DB_PREFIX . 'shop_inform_product_available');
+            $alerts[]  = \Nails\Admin\Nav::alertObject($numAlertsAll, 'info', 'Product Availability Alerts');
+
+            //  Get notifications in the last week, add an alert count
+            $ci->db->where('created', '> ADDDATE(NOW(), INTERVAL -1 WEEK');
```

Comment:
Should probably be written as so, also, note missing closing parenthesis:

     $ci->db->where('created >', 'ADDDATE(NOW(), INTERVAL -1 WEEK)', false);